Moderation-Policy.md

```diff
@@ -209,6 +209,11 @@ The Moderation Team may, at their discretion, choose to enable GitHub's
 [Temporary Interaction Limits][] on any GitHub repository in the Node.js GitHub
 Organization.
 
+Any Collaborator may request that the Moderation Team enable the Temporary
+Interaction Limits by posting an issue to the moderation repository. If the
+Moderation Team choose not to do so, then a comment explaining why that
+decision was made should be added to the moderation repository thread.
```

Comment:
I think this ought to be "must be added." Otherwise, it leaves a gap where the Moderation Team decides not to, and then never posts a decision to the thread. This would not look good.